src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/github-connector.css

```diff
@@ -0,0 +1,79 @@
+.create-github-api-key-container {
```

Comment:
I don't understand the relation with "upcoming UI work". Ok, if you prefer use `<div class="github-branch-source">`. Identifiers, well used, provide interesing thinks.

:ant: